frontend/src/i18n/messages/en/index.ts

```diff
@@ -333,15 +333,22 @@ export const messages = {
     list: "<ul><li>Hear about upcoming milestones</li><li>Be the first to know when we launch new code</li><li>Test out new features and functionalities</li><li>Participate in usability tests and other user research efforts</li><li>Learn about ways to provide feedback </li></ul>",
     disclaimer:
       "The Simpler.Grants.gov email subscriptions are powered by the Sendy data service. Personal information is not stored within Simpler.Grants.gov.",
+    form: {
+      name: "First Name",
+      lastName: "Last Name",
+      email: "Email",
+      req: "required",
+      opt: "optional",
+      button: "Subscribe",
+    },
     errors: {
-      missing_name: "Enter your first name.",
-      missing_email: "Enter your email address.",
+      missing_name: "Please enter a name.",
+      missing_email: "Please enter an email address.",
       invalid_email:
         "Enter an email address in the correct format, like name@example.com.",
-      already_subscribed:
-        "<email_address/> is already subscribed. If you’re not seeing our emails, check your spam folder and add no-reply@grants.gov to your contacts, address book, or safe senders list. If you continue to not receive our emails, contact <email>simpler@grants.gov</email>.",
-      sendy:
-        "Sorry, an unexpected error in our system occured when trying to save your subscription. If this continues to happen, you may email <email>simpler@grants.gov</email>. Error: <sendy_error/>",
+      server:
+        "Failed to subscribe, due to a server error. Please try again later.",
+      already_subscribed: "This email address has already been subscribed.",
```

Comment:
❓

Reply:
Just curious where the impetus for changing "<email_address/> is already subscribed. If you’re not seeing our emails, check your spam folder and add no-reply@grants.gov to your contacts, address book, or safe senders list. If you continue to not receive our emails, contact <email>simpler@grants.gov</email>." to "This email address has already been subscribed." is coming from exactly since I didn't see it in any requirements. Very possible I just missed that req somewhere

Reply:
Sorry, I should have been more verbose. Not sure where that came from myself. @emilycnava ?

Reply:
It wasn't a req, just a choice I made due to the positioning of the error message being different and long messages looking weird. Also the email var insertion code was broken with the old approach.

Old:
<img width="841" alt="Screenshot 2024-10-28 at 3 07 53 PM" src="https://github.com/user-attachments/assets/329c48c4-2ebc-492c-94a1-b2fd156d04ef">

New:
<img width="703" alt="Screenshot 2024-10-28 at 3 11 41 PM" src="https://github.com/user-attachments/assets/047fe150-0623-491b-a455-913cab49ab1f">

Reply:
I'm ok with this since it's fixing a bug with the formatting, but we should probably get sign off from design / content on the final text